cfme/services/requests.py

```diff
@@ -40,17 +40,27 @@ def __attrs_post_init__(self):
     def wait_for_request(self, num_sec=1800, delay=20):
         def _finished():
             self.rest.reload()
-            return self.rest.request_state.title() in self.REQUEST_FINISHED_STATES
+            return self.rest.request_state.title() in self.REQUEST_FINISHED_STATES and \
```

Comment:
Required: Please use parentheses for line continuation
The same for later occurrences